pkg/transform/oauth/oauth_test.go

```diff
@@ -106,6 +106,6 @@ func TestGenYAML(t *testing.T) {
 	CRD := crd.GenYAML()
 	expectedYaml, _ := ioutil.ReadFile("testdata/expected-bulk-test-masterconfig-oauth.yaml")
 
-	assert.Equal(t, 10, len(manifests))
+	assert.Equal(t, 9, len(manifests))
```

Comment:
Sorry but I can't see why the number has decreased?

Reply:
One of secrets here was empty

Reply:
Thanks for fixing this. I couldn't understand why the empty secret was being generated. I definitely saw it yesterday when I was working on the oauth transform test.